src/libs/metadata_view.c

```diff
@@ -93,66 +96,73 @@ enum
   md_size
 };
 
+static gchar *_labels[md_size];
 static gchar *_md_labels[md_size];
 
 /* initialize the labels text */
 static void _lib_metatdata_view_init_labels()
 {
   /* internal */
-  _md_labels[md_internal_filmroll] = _("filmroll");
-  _md_labels[md_internal_imgid] = _("image id");
-  _md_labels[md_internal_groupid] = _("group id");
-  _md_labels[md_internal_filename] = _("filename");
-  _md_labels[md_internal_version] = _("version");
-  _md_labels[md_internal_fullpath] = _("full path");
-  _md_labels[md_internal_local_copy] = _("local copy");
-  _md_labels[md_internal_import_timestamp] = _("import timestamp");
-  _md_labels[md_internal_change_timestamp] = _("change timestamp");
-  _md_labels[md_internal_export_timestamp] = _("export timestamp");
-  _md_labels[md_internal_print_timestamp] = _("print timestamp");
+  _labels[md_internal_filmroll] = "filmroll";
```

Comment:
You'll want to mark all of these strings for translation using for example `N_("filmroll")`, otherwise the call to `_(_labels[i])` below won't be able to find any translations.

Reply:
I' think I've already experienced the issue you talk about. But as the screenshots show the translation in French is done. I'm a bit lost here ...

Reply:
Ah, but that's because the translations haven't been updated yet. But when this gets merged and the new list of translatable strings gets generated, all the existing translations for these strings will get purged, because it will look like they are no longer needed.

Translations are tricky :-)

Reply:
So, I should use _label[i] = N_() to set the untranslated strings, and then _md_label[i] = get_text(_label[i]) to get the translated one. Is that correct ?

Reply:
`_labels[md_internal_filmroll] = N_("filmroll");`

you can use 
`_md_label[i] = _(_label[i]);` 
instead of 
`_md_label[i] = gettext(_label[i]);` 
it is the same.